<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ukui_NoteBook</class>
 <widget class="QWidget" name="ukui_NoteBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NoteBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
border:1px solid rgba(255, 255, 255, 0.05);</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>380</width>
     <height>204</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(165, 165, 165);
border-radius:4px;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>380</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(165, 165, 165);</string>
   </property>
   <widget class="QToolButton" name="ukui_NoteFont">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/SVG/font.svg);
border-width:0;
border-style:outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ukui_NoteJpg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/SVG/exportJPG.svg);
border-width:0;
border-style:outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ukui_NotePdf">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/SVG/exportPDF.svg);
border-width:0;
border-style:outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ukui_NoteDate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/SVG/insertTime.svg);
border-width:0;
border-style:outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="ukui_NoteDel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/SVG/delete.svg);
border-width:0;
border-style:outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ukui_NoteSave">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(61,107,229,1);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ukui_NoteSaveAs">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(61,107,229,1);
border-radius:4px;</string>
    </property>
    <property name="text">
     <string>SaveAs</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ukui_NoteNew">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ukui_NoteClose">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
